dist/adf-widget-scm.js

```diff
@@ -183,17 +183,22 @@ angular.module('adf.widget.scm', ['adf.provider', 'chart.js', 'ngSanitize', 'btf
-$templateCache.put("{widgetsPath}/scm/src/markdownPreview/view.html","<style>\n  div.markdownContent{\n    overflow: auto;\n    width: 100%;\n  }\n</style><div class=\"alert alert-info\" ng-if=!vm.fileContent>Please configure a specific file</div><div ng-if=vm.fileContent btf-markdown=vm.fileContent class=markdownContent></div><div class=\"alert alert-danger\" ng-if=\"vm.fileContent.status == 500 || vm.fileContent.status == 404\"><b>Error {{vm.fileContent.status}}</b> Markdown-File not found. Please check your configuration and try again.</div>");}]);
+$templateCache.put("{widgetsPath}/scm/src/markdownPreview/edit.html","<form role=form><div class=form-group><p><label for=repository>Repository</label> <span class=\"grey glyphicon glyphicon-info-sign ng-scope\" tabindex=0 ng-focus=\"repositoryTooltipIsOpen = !repositoryTooltipIsOpen\" tooltip-trigger=outsideClick tooltip-is-open=repositoryTooltipIsOpen ng-mouseenter=\"repositoryTooltipIsOpen = !repositoryTooltipIsOpen\" uib-tooltip-html=repositoryTooltip ng-mouseover=\"tooltipIsOpenRepo = !tooltipIsOpenRepo\" tooltip-placement=top aria-label=\"A valid URL to your Jenkins server must be stored here\"></span><select name=repository id=repository class=form-control ng-model=config.repository ng-init=vm.getBranchesByRepositoryId(config.repository) ng-change=vm.getBranchesByRepositoryId(config.repository)><option ng-repeat=\"repository in vm.repositories | orderBy: \'name\'\" value={{repository.id}}>{{repository.name}}</option></select></p><p ng-if=vm.branches><label for=branch>Branch</label> <span class=\"grey glyphicon glyphicon-info-sign ng-scope\" tabindex=0 ng-focus=\"branchTooltipIsOpen = !branchTooltipIsOpen\" tooltip-trigger=outsideClick uib-tooltip-html=branchTooltip ng-mouseover=\"branchTooltipIsOpen = !branchTooltipIsOpen\" tooltip-placement=top tooltip-is-open=branchTooltipIsOpen aria-label=\"A valid URL to your Jenkins server must be stored here\"></span><select name=branch id=branch class=form-control ng-model=config.branch><option ng-repeat=\"branch in vm.branches| orderBy: \'name\'\" value={{branch.id}}>{{branch.name}}</option></select></p><p ng-if=vm.branches><label for=path>Path to Markdown File</label> <span class=\"grey glyphicon glyphicon-info-sign ng-scope\" tabindex=0 ng-focus=\"pathToMarkdownTooltipIsOpen = !pathToMarkdownTooltipIsOpen\" tooltip-trigger=outsideClick uib-tooltip-html=pathToMarkdownTooltip ng-mouseover=\"pathToMarkdownTooltipIsOpen = !pathToMarkdownTooltipIsOpen\" tooltip-is-open=pathToMarkdownTooltipIsOpen tooltip-placement=top aria-label=\"A valid path to a markdown file must be stored here\"></span> <input type=text class=form-control id=path ng-model=config.path required></p></div></form>");
```

Comment:
Is the URL of the Jenkins server the right default message?
Please refer to voice & tone when writing messages. “A valid path to a markdown file must be stored here” might be rewritten to “Enter the path to a markdown file”.
Add information if the format of the path has to follow a special format (no leading / for example)